### ISSUE TITLE
Changed repository name of one of the composer.lock included packages.

### ISSUE THREAD

**Issue by miloszowi** [magento/magento2#33920]
In the `composer.lock` file in current Magento 2 [codebase](https://github.com/magento/magento2/blob/2.4-develop/composer.lock#L7769) we can find module named `allure-framework/allure-php-api` which has source on `<https://github.com/allure-framework/allure-php-commons.git>`

According to this [commit](https://github.com/allure-framework/allure-php-api/commit/3e4ccbf48aedda6730227034be3505475b868e90) repository name has been changed leading to crash every Magento 2 application failing caused by not existing source.

 
```java
  - Installing allure-framework/allure-php-api (1.1.8): Downloading (connecting...).................Downloading (0%) ............ Failed to download allure-framework/allure-php-api from dist: The "https://codeload.github.com/allure-framework/allure-php-commons/legacy.zip/<hash>" file could not be downloaded (HTTP/1.1 404 Not Found) Now trying to download from source - Installing allure-framework/allure-php-api (1.1.8): Failed to update https://github.com/allure-framework/allure-php-commons.git in cache, package installation for allure-framework/allure-php-api might fail. Cloning <hash> from cache <hash> is gone (history was rewritten?)
```

**Comment by ihor-sviziev** [comment]
Hi @miloszowi,
The internal team already fixed this issue in the 2.4-develop branch in https://github.com/magento/magento2/commit/2670c4b531b3a28840857f5bdfec8de18eb1ffa3.

For those who experience this issue on their projects - please run the following command: 
```
composer update allure-framework/allure-php-api
```

I'm closing this issue.

**Comment by canon1ij** [comment]
Well, Well, Well.... You Know 
Cricut machine makes the process of cutting materials easy. Once your machine is set up, you can give a command to your machine through your computer and start making crafts quickly and easily. The process of setting up a Cricut machine is simple. You can download and install the Cricut Design Space for your Windows and Mac. To do so, you can head over to the official site [cricut.com/setup](https://ny-cricut.com/). Once you have downloaded and installed the software, initiate the process of setting up the craft machine.
